rust/main/agents/relayer/src/test_utils/mock_ism.rs

```diff
@@ -64,10 +64,7 @@ impl InterchainSecurityModule for MockInterchainSecurityModule {
             .lock()
             .unwrap()
             .pop_front()
-            .expect(&format!(
-                "No mock dry_run_verify response set {}",
-                self.address
-            ))
+            .unwrap_or_else(|| panic!("No mock dry_run_verify response set {}", self.address))
```

Comment:
nit: any reason we prefer this over expect?

Reply:
this was just clippy 😄

Reply:
ah got it :+1: